rpcs3/Emu/RSX/GL/GLFragmentProgram.cpp

```diff
@@ -68,13 +68,18 @@ void GLFragmentDecompilerThread::insertConstants(std::stringstream & OS)
 
 	}
 
+	OS << "layout(std140, binding = 2) uniform FragmentConstantsBuffer" << std::endl;
+	OS << "{" << std::endl;
 	for (ParamType PT : m_parr.params[PF_PARAM_UNIFORM])
```

Comment:
I dunno what happen there, I added the comment on a line that had something like (for PT : list) {}